eng/MPack.props

```diff
@@ -1,5 +1,8 @@
 <Project>
   <PropertyGroup>
     <ArtifactsMPackDir>$(ArtifactsDir)MPack\$(Configuration)\</ArtifactsMPackDir>
+    <_MonoDevelopPath>$(MSBuildThisFileDirectory)..\..\monodevelop\</_MonoDevelopPath>
+    <_MDLocalConfigDir>$(_MonoDevelopPath)local-config\</_MDLocalConfigDir>
+    <_VSMXamarinExtensionsPathBundled>$([System.IO.Path]::GetFullPath($(MSBuildThisFileDirectory)..\..\md-addins\external-addins\))</_VSMXamarinExtensionsPathBundled>
```

Comment:
Yeah I'd really prefer to not see setup-specific information check in.

Reply:
See above, more VSmac-specific than setup-specific.